app/src/main/java/com/kickstarter/libs/TrackingClient.kt

```diff
@@ -124,4 +124,17 @@ abstract class TrackingClient(
             cm.getNetworkCapabilities(it)?.hasTransport(NetworkCapabilities.TRANSPORT_WIFI)
         } ?: false
     }
+
+    override fun sessionForceDarkMode(): Boolean {
```

Comment:
just an fyi, no need to act on it. This method could probably just be a private method, not added to the interface. 
The trackingclienttype interface could use some refactoring (@leighdouglas might remember some conversations we had around the topic), the interface should probably contain only generic action methods (tack, identify, reset, isEnabled ... ) that can be applied to any tracking system (Segment, Google Analytics or whatever tracking third party we might want). The rest are properties and not necessarily behavioural